configs/client/config.json

```diff
@@ -11,7 +11,7 @@
   },
   "TORII_API_URL": "http://127.0.0.1:8080/",
   "TORII_TELEMETRY_URL": "http://127.0.0.1:8180/",
-  "TRANSACTION_TIME_TO_LIVE_MS": 100000,
+  "TRANSACTION_TIME_TO_LIVE_MS": null,
```

Comment:
Probably should be returned.

Reply:
I'm closing this in favor of discussion [here](https://github.com/hyperledger/iroha/pull/3485#discussion_r1216936773)